modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb

```diff
@@ -17,6 +17,8 @@ class ClaimBlueprint < Blueprinter::Base
         field :status
         field :submitter_application_code
         field :submitter_role_code
+        field :end_product_code
+        # field :close_date
```

Comment:
It looks like this ticket wanted us to begin showing `closeDate`.
So we'll need to uncomment this and add the logic required to extract that value and begin providing it in the responses.